sdk/cosmos/azure-cosmos/azure/cosmos/_cosmos_client_connection.py

```diff
@@ -175,7 +180,6 @@ def __init__(
             proxy = host if url.port else host + ":" + str(self.connection_policy.ProxyConfiguration.Port)
             proxies.update({url.scheme: proxy})
 
-        self._user_agent = _utils.get_user_agent()
```

Comment:
Was this not being used?

Reply:
It is, I simply moved it when I was applying my older approach.